features/items.feature

```diff
@@ -225,3 +225,53 @@ Feature:  items
       | Number | Number1 |
       | String | String1 |
       | Player | Player1 |
+
+  Scenario: Item returns nil for an unlinked item
+    Given items:
+      | type | name |
+      | Number | Number1 |
+    And code in a rules file
+      """
+      logger.info("Thing: #{Number1.linked_thing}")
+      """
+    When I deploy the rules file
+    Then It should log "Thing: " within 5 seconds
```

Comment:
You should probably use inspect and check for nil in the log line. Otherwise this test will always match regardless of what's logged